e2e/cypress/integration/enterprise/system_console/channel_moderation_spec.js

```diff
@@ -35,24 +59,876 @@ const enableAllChannelModeratedPermissions = () => {
+            saveConfig();
+
+            visitAutemChannel('user-1');
+
+            cy.findByTestId('post_textbox').clear().type('@');
+
+            // * Ensure that @here, @all, and @channel do not show up in the autocomplete list
+            cy.findAllByTestId('mentionSuggestion_here').should('not.exist');
+            cy.findAllByTestId('mentionSuggestion_all').should('not.exist');
+            cy.findAllByTestId('mentionSuggestion_channel').should('not.exist');
+
+            // * When you type @all, @enter, and @channel make sure that a system message shows up notifying you nothing happened.
+            postChannelMentionsAndVerifySystemMessageExist();
+        });
+
+        it('Confirm sending notifications while using special channel mentions', () => {
+            // # Visit Channel page and Search for the channel.
+            visitAutemChannelConfigPage();
+            disableChannelModeratedPermission(checkboxesTitleToIdMap.CHANNEL_MENTIONS_MEMBERS);
+            saveConfig();
+
+            // # Set @channel and @all confirmation dialog to true
+            cy.visit('admin_console/environment/notifications');
+            cy.findByTestId('TeamSettings.EnableConfirmNotificationsToChanneltrue').check();
+            saveConfig();
+
+            // # Visit autem channel
+            visitAutemChannel('user-1');
+
+            // * Type at all and enter that no confirmation dialogue shows up
+            cy.findByTestId('post_textbox').clear().type('@all{enter}');
+            cy.get('#confirmModalLabel').should('not.exist');
+
+            // * Type at channel and enter that no confirmation dialogue shows up
+            cy.findByTestId('post_textbox').clear().type('@channel{enter}');
+            cy.get('#confirmModalLabel').should('not.exist');
+
+            // * Type at here and enter that no confirmation dialogue shows up
+            cy.findByTestId('post_textbox').clear().type('@here{enter}');
+            cy.get('#confirmModalLabel').should('not.exist');
+        });
+    });
+
+    describe('MM-23102 - Higher Scoped Scheme', () => {
+        const createNewChannel = (channelName, createAs) => {
+            cy.apiLogin(createAs);
+            cy.visit('/ad-1/channels');
+            cy.get('#createPublicChannel').click();
+            cy.get('#newChannelName').clear().type(channelName);
+            cy.get('#submitNewChannel').click();
+        };
+
+        it('Effect of changing System Schemes on a Channel for which Channel Moderation Settings was modified', () => {
+            // # Visit Channel page and Search for the channel.
+            visitAutemChannelConfigPage();
+            disableChannelModeratedPermission(checkboxesTitleToIdMap.MANAGE_MEMBERS_MEMBERS);
+            saveConfig();
+
+            // # check the channel mentions option for guests and save
+            enableChannelModeratedPermission(checkboxesTitleToIdMap.MANAGE_MEMBERS_MEMBERS);
+            saveConfig();
```

Comment:
When we disable & enable the same config, the channel scheme gets removed and is treated as channel permission is not modified. Hence can you modify the test to disable some Channel Permission like Special Mentions before going to System Scheme. This way, it is ensured that the channel scheme is present.

Reply:
Done! I now disable manage members and channel mentions for members then save the config and then re-enable manage members for members before going to the system scheme.